cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
And as I said it appear that same visibility settings in pure Xcode project behave differently. I guess somebody has to go on setting hunt again :|